app/components/team/Row.js

```diff
@@ -41,7 +41,7 @@ class TeamRow extends React.PureComponent {
       <Panel.RowLink key={this.props.team.id} to={`/organizations/${this.props.team.organization.slug}/teams/${this.props.team.slug}`}>
         <div className="flex flex-stretch items-center line-height-1" style={{ minHeight: '3em' }}>
           <div className="flex-auto">
-            <div className="m0 flex items-center">
+            <div className="m0 flex items-center mb1">
```

Comment:
This should probably be done with `line-height` instead, as this padding also applies to teams which don’t have a description, causing their names to be off-centre.